cilium/cmd/status.go

```diff
@@ -69,29 +56,51 @@ func statusDaemon(cmd *cobra.Command, args []string) {
 			if nm.Lost != 0 || nm.Unknown != 0 {
 				fmt.Fprintf(w, "\t%d events lost, %d unknown notifications\n", nm.Lost, nm.Unknown)
 			}
-		} else {
-			fmt.Fprintf(w, "NodeMonitor:\tDisabled\n")
 		}
+		if sr.Cilium != nil && sr.Cilium.State != models.StatusStateOk {
+			os.Exit(1)
+		}
+	}
 
-		if sr.IPAM != nil {
-			fmt.Fprintf(w, "Allocated IPv4 addresses:\n")
-			for _, ipv4 := range sr.IPAM.IPV4 {
-				fmt.Fprintf(w, " %s\n", ipv4)
+}
 
-			}
-			fmt.Fprintf(w, "Allocated IPv6 addresses:\n")
-			for _, ipv6 := range sr.IPAM.IPV6 {
-				fmt.Fprintf(w, " %s\n", ipv6)
-			}
+func printStatus(w *tabwriter.Writer, sr *models.StatusResponse) {
```

Comment:
+1 to making this a function

Reply:
Will drop this function since there is [FormatStatusResponse](https://godoc.org/github.com/cilium/cilium/pkg/client#FormatStatusResponse) now.